### ISSUE TITLE
Зеркала гос-сайтов с возможностью комментирования

### ISSUE THREAD

**Issue by lain-dono** [russian-plan/first#1]
Как на счёт запилить зеркальные версии для gov-сайтов с более удобной навигацией и возможностью комментирования всего этого. Алсо неплохо было бы превратить растровые pdf с оных в чистый текст, дабы по всему этому можно было проводить поиск и прочий автоматический анализ. Ну и запилить гайды по пользованию всем этим зоопарком гос-сайтов.

Окей, я помню всю шумиху по поводу принятия всех этих законов. Я может около пары десятков человек сподобил проголосовать на бесполезном(как в последствии оказалось) roi и даже на митинги ходил. Честно говоря разовые действия в интернете бесполезны (политиканы не умеют в интернет и стремятся его уничтожить), а легальные митинги с каждым днём всё менее легальны.

Надо что-то более осмысленное пилить.

**Comment by imShara** [comment]
Эх, были проекты год/два назад с наивной надеждой на возможность воздействия на эту систему. Но сейчас, как по мне, единственное что имеет смысл разработчикам делать на этом поприще - так это совершенно новую систему управления государством, чтобы не встать на те же грабли после революции. И уезжать, конечно.

**Comment by M1xey** [comment]
"И уезжать, конечно."
Куда?

**Comment by dimon-durak** [comment]
>  И уезжать, конечно.
Зачем?

21 апреля 2015 г., 2:55 пользователь imShara <notifications@github.com>
написал:

**Comment by imShara** [comment]
> Куда?

По вкусу

> Зачем?

Если нет на то причин, то незачем

**Comment by kisik21** [comment]
@M1xey в Японию, например. Там спокойней.

**Comment by M1xey** [comment]
kisik21, пока.

**Comment by kisik21** [comment]
Скоро вообще третья и четвертая мировые войны начнутся.

С уважением, ヴ｡
Отправлено с моего Windows Phone
________________________________

**Comment by M1xey** [comment]
Нет, я не на это намекал. Такой сценарий развития И-нета запросто подхватят другие страные, если посчитают такой ход событий эффективным. Пару лет поживёшь спокойно, а затем снова придётся столкнуться с этой проблемой.

**Comment by kisik21** [comment]
Тогда лучше улетать в Тор. Нафиг.

С уважением, ヴ｡
Отправлено с моего Windows Phone
________________________________
